Changing text and link of GitHub ribbon.

gvwilson opened the pull request:
This addresses #125 by changing the text of the GitHub ribbon to "Find us on GitHub"
and changing the link to point to the organization `https://github.com/swcarpentry`.

gvwilson committed:
Changing text and link of GitHub ribbon.

This addresses #125 by changing the text of the GitHub ribbon to "Find us on GitHub"
and changing the link to point to the organization `https://github.com/swcarpentry`.

wking commented:
On Tue, Jan 27, 2015 at 02:28:01AM -0800, Greg Wilson wrote:

> This addresses #125 by changing the text of the GitHub ribbon to
> "Find us on GitHub" and changing the link to point to the
> organization `https://github.com/swcarpentry`.

I though we wanted to populate the URL from _config.yml's lesson_repo?
Since many (the majority?) of workshop repositories will be under the
lead instructor's namespace [1](https://github.com/swcarpentry/workshop-template/tree/aefe7db4d6715cfd617c0d4943b53645770efc24#creating-a-repository), it doesn't seem right to point folks
at swcarpentry.

gvwilson committed:
Renaming `_config.yml` values and using them in web pages.

1.  Changed `lesson_repo` and `lesson_site` to `workshop_repo` and `workshop_site` in `_config.yml`.
2.  Updated `_includes/footer.html` to match.
3.  Updated docs in `README.md` to match.

Rationale is that this is the workshop repo, not a lesson repo, so we shouldn't use `lesson_` as a prefix for variables.

gvwilson committed:
Make the GitHub ribbon point to this workshop's GtiHub repository using a variable from `_config.yml`.

rgaiacs commented:
> I though we wanted to populate the URL from _config.yml's lesson_repo?

This is the workshop repo.

:+1: to merge this.

gvwilson commented:
@r-gaia-cs @wking I've changed the variable names to `workshop_` instead of `lesson_` in `_config.yml` and am now using the variable in the GitHub ribbon - better?

rgaiacs commented:
@gvwilson :+1:

wking commented:
On Thu, Jan 29, 2015 at 03:06:19AM -0800, Greg Wilson wrote:

> I've changed the variable names to `workshop_` instead of `lesson_`
> in `_config.yml` and am now using the variable in the GitHub ribbon

Looks good to me.

wking commented:
On Thu, Jan 29, 2015 at 09:08:13AM -0800, W. Trevor King wrote:

> Thu, Jan 29, 2015 at 03:06:19AM -0800, Greg Wilson:
> 
> > I've changed the variable names to `workshop_` instead of `lesson_`
> > in `_config.yml` and am now using the variable in the GitHub ribbon
> 
> Looks good to me.

Actually, it looks like you missed some lesson_ → workshop_ fixes:

  $ git grep lesson_ origin/pr/148
  origin/pr/148:CUSTOMIZATION.md:and change the URLS called `lesson_repo` and `lesson_site`

Once you fix those it looks good to me ;).

gvwilson committed:
Fixing two more references to lesson_
